diff --git a/node_modules/typedoc-plugin-markdown/dist/resources/helpers/if-show-type-hierarchy.js b/node_modules/typedoc-plugin-markdown/dist/resources/helpers/if-show-type-hierarchy.js
index 06eb8bb..254addb 100644
--- a/node_modules/typedoc-plugin-markdown/dist/resources/helpers/if-show-type-hierarchy.js
+++ b/node_modules/typedoc-plugin-markdown/dist/resources/helpers/if-show-type-hierarchy.js
@@ -8,4 +8,4 @@ function ifShowTypeHierarchy(options) {
         ? options.fn(this)
         : options.inverse(this);
 }
-exports.ifShowTypeHierarchy = ifShowTypeHierarchy;
+exports.ifShowTypeHierarchy = () => ''
diff --git a/node_modules/typedoc-plugin-markdown/dist/resources/partials/member.sources.hbs b/node_modules/typedoc-plugin-markdown/dist/resources/partials/member.sources.hbs
index a999ee8..e69de29 100755
--- a/node_modules/typedoc-plugin-markdown/dist/resources/partials/member.sources.hbs
+++ b/node_modules/typedoc-plugin-markdown/dist/resources/partials/member.sources.hbs
@@ -1,55 +0,0 @@
-{{#if implementationOf}}
-
-#### Implementation of
-
-{{#with implementationOf}}
-
-{{typeAndParent}}
-
-{{/with}}
-
-{{/if}}
-
-{{#if inheritedFrom}}
-
-#### Inherited from
-
-{{#with inheritedFrom}}
-
-{{{typeAndParent}}}
-
-{{/with}}
-
-{{/if}}
-
-{{#if overwrites}}
-
-#### Overrides
-
-{{#with overwrites}}
-
-{{typeAndParent}}
-
-{{/with}}
-
-{{/if}}
-
-{{#if sources}}
-
-#### Defined in
-
-{{#each sources}}
-
-{{#if url}}
-
-[{{fileName}}:{{line}}]({{url}})
-
-{{else}}
-
-{{fileName}}:{{line}}
-
-{{/if}}
-
-{{/each}}
-
-{{/if}}
\ No newline at end of file
diff --git a/node_modules/typedoc-plugin-markdown/dist/resources/partials/title.hbs b/node_modules/typedoc-plugin-markdown/dist/resources/partials/title.hbs
index ba8e5be..d9627dc 100644
--- a/node_modules/typedoc-plugin-markdown/dist/resources/partials/title.hbs
+++ b/node_modules/typedoc-plugin-markdown/dist/resources/partials/title.hbs
@@ -3,5 +3,3 @@
 # {{{reflectionTitle true}}}
 
 {{/ifShowPageTitle}}
-
-{{{reflectionPath}}}
\ No newline at end of file
diff --git a/node_modules/typedoc-plugin-markdown/dist/resources/templates/reflection.hbs b/node_modules/typedoc-plugin-markdown/dist/resources/templates/reflection.hbs
index ce29bc5..3301cfe 100755
--- a/node_modules/typedoc-plugin-markdown/dist/resources/templates/reflection.hbs
+++ b/node_modules/typedoc-plugin-markdown/dist/resources/templates/reflection.hbs
@@ -36,26 +36,6 @@
 
 {{/ifShowTypeHierarchy}}
 
-{{#if model.implementedTypes}}
-
-## Implements
-
-{{#each model.implementedTypes}}
-- {{{type}}}
-{{/each}}
-
-{{/if}}
-
-{{#if model.implementedBy}}
-
-## Implemented by
-
-{{#each model.implementedBy}}
-- {{{type}}}
-{{/each}}
-
-{{/if}}
-
 {{#if model.signatures}}
 
 ## Callable
