diff --git a/node_modules/typedoc-plugin-markdown/dist/libs/utils/escape-chars.js b/node_modules/typedoc-plugin-markdown/dist/libs/utils/escape-chars.js
index e29ab44..38f70d5 100644
--- a/node_modules/typedoc-plugin-markdown/dist/libs/utils/escape-chars.js
+++ b/node_modules/typedoc-plugin-markdown/dist/libs/utils/escape-chars.js
@@ -5,7 +5,7 @@ export function escapeChars(str) {
         .replace(/{/g, '\\{')
         .replace(/}/g, '\\}')
         .replace(/_/g, '\\_')
-        .replace(/`/g, '\\`')
+        // .replace(/`/g, '\\`')
         .replace(/\|/g, '\\|')
         .replace(/\[/g, '\\[')
         .replace(/\]/g, '\\]')
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/helpers/get-project-name.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/helpers/get-project-name.js
index 1c71dae..c031a54 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/helpers/get-project-name.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/helpers/get-project-name.js
@@ -4,6 +4,6 @@ export function getProjectName(stringWithPlaceholders, page, includeVersion = tr
         .replace('{version}', includeVersion && page.project.packageVersion
         ? `v${page.project.packageVersion}`
         : '')
-        .replace(/\s+/g, ' ')
+        // .replace(/\s+/g, ' ')
         .trim();
 }
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.declarationTitle.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.declarationTitle.js
index 17444bd..0438248 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.declarationTitle.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.declarationTitle.js
@@ -63,5 +63,5 @@ export function declarationTitle(model) {
         md.push(';');
     }
     const result = md.join('');
-    return useCodeBlocks ? codeBlock(result) : `> ${result}`;
+    return useCodeBlocks ? codeBlock(result) : `• ${result}`;
 }
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.hierarchy.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.hierarchy.js
index b1d7fd3..b23a8da 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.hierarchy.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.hierarchy.js
@@ -1,6 +1,7 @@
 import { heading, unorderedList } from '../../../libs/markdown/index.js';
 import { i18n } from 'typedoc';
 export function hierarchy(model, options) {
+    return '';
     const md = [];
     const getHierarchy = (hModel) => {
         const parent = !hModel.isTarget
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.inheritance.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.inheritance.js
index 29eaa2b..c0c0f6c 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.inheritance.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.inheritance.js
@@ -1,6 +1,7 @@
 import { heading } from '../../../libs/markdown/index.js';
 import { i18n } from 'typedoc';
 export function inheritance(model, options) {
+    return '';
     const md = [];
     if (model.implementationOf) {
         if (options.headingLevel !== -1) {
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.memberWithGroups.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.memberWithGroups.js
index 3b9cc6a..b6e5fd8 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.memberWithGroups.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.memberWithGroups.js
@@ -42,7 +42,7 @@ export function memberWithGroups(model, options) {
             }));
         }
     }
-    if (model.implementedTypes?.length) {
+    if (false) {
         md.push(heading(options.headingLevel, i18n.theme_implements()));
         md.push(unorderedList(model.implementedTypes.map((implementedType) => this.partials.someType(implementedType))));
     }
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.signatureTitle.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.signatureTitle.js
index cbc03e8..5fe0813 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.signatureTitle.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/member.signatureTitle.js
@@ -38,5 +38,5 @@ export function signatureTitle(model, options) {
         md.push(';');
     }
     const result = md.join('');
-    return useCodeBlocks ? codeBlock(result) : `> ${result}`;
+    return useCodeBlocks ? codeBlock(result) : `▸ ${result}`;
 }
diff --git a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/page.pageTitle.js b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/page.pageTitle.js
index 1df516f..9ce6394 100644
--- a/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/page.pageTitle.js
+++ b/node_modules/typedoc-plugin-markdown/dist/theme/context/partials/page.pageTitle.js
@@ -117,6 +117,6 @@ function getFromString(textContent, config) {
         .replace('{codeKeyword}', config.codeKeyword ?? '')
         .replace('{group}', config.group ?? '')
         .replace(/``/g, '')
-        .replace(/\s+/g, ' ')
+        // .replace(/\s+/g, ' ')
         .trim();
 }
